portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,77 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*; 
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private final ArrayList<String> listData = new ArrayList<String>();
+    private final ArrayList<String> comments = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+   
+    listData.add("Hello!"); 
+    listData.add("How are you?");
+    listData.add("Great weather today!");
+
+    //converting data to JSON
+    String json = convertToJson(listData);
+    
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+  }
+
+  private String convertToJson(ArrayList<String> data) {
+    String json = "";
+    json += data.get(0);
+    json += ", ";
+    json += data.get(1);
+    json += ", ";
+    json += data.get(2);
+    json += "";
```

Comment:
Nit: You could probably generalize this with a for-loop to work for ArrayLists of any size using less lines of code.